Jint/Runtime/JavaScriptException.cs

```diff
@@ -12,8 +12,9 @@ namespace Jint.Runtime
     public class JavaScriptException : JintException
     {
         private string? _callStack;
+		//private JavaScriptException jsException;
 
-        public JavaScriptException(ErrorConstructor errorConstructor) : base("")
+		public JavaScriptException(ErrorConstructor errorConstructor) : base("")
```

Comment:
ah - yeah - sorry - I wasn't sure if your project preferred tabs or spaces for indenting - I actually didn't realize visual studio had used spaces - so I just updated it with tabs.

Visual Studio does this weird thing where by default for C# it does whatever your preference is (i.e. for me it is tabs - not spaces) - but if it sees spaces for indenting in the file - it will use that instead of whatever you've set for your preference.